test/Interop/Cxx/function/Inputs/EmitCalledFunction.h

```diff
@@ -0,0 +1,7 @@
+inline int bar() {
+  return 42;
+}
+
+inline int foo() {
+  return bar();
+}
```

Comment:
Please use more descriptive names than foo and bar.

Reply:
Done.

While I was doing this, it occurred to me that I should also add a function that isn't called from Swift at all (whether directly or indirectly) and test that it isn't emitted.